deeplake/core/chunk/base_chunk.py

```diff
@@ -559,3 +567,12 @@ def _fill_empty_shapes(self, shape, num_samples):
     @property
     def is_empty_tensor(self):
         return len(self.tensor_meta.max_shape) == 0 and len(self.data_bytes) == 0
+
+    def _text_sample_to_byte_string(self, sample):
+        try:
+            return sample.encode("utf-8")
+        except AttributeError:
+            try:
+                return sample.tolist().encode("utf-8")
+            except AttributeError:  # None
+                return b""
```

Comment:
Using try-except blocks instead of isinstance checks because its faster to ask for forgiveness than permission. We cover 3 cases here - str,  ndarray with shape == () and None.